R/loadFromMothur.R

```diff
@@ -0,0 +1,206 @@
+#' Import Mothur results to \code{TreeSummarizedExperiment}
+#'
+#' @param featureTableFile a single \code{character} value defining the file
```

Comment:
Description test is missing. Please add it, even though it might be only one sentence

Reply:
Added